db/migrate/20160114011126_create_indices.rb

```diff
@@ -0,0 +1,13 @@
+class CreateIndices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :indices do |t|
+      t.datetime :finalized_at
+      t.string :awrvi_version
```

Comment:
I think this should reference a root node of Category, which corresponds to the AWRVI version.

cc @teknofire